android/modules/ui/src/java/ti/modules/titanium/ui/AlertDialogProxy.java

```diff
@@ -23,8 +23,11 @@
 		TiC.PROPERTY_BUTTON_NAMES,
 		TiC.PROPERTY_CANCEL,
 		TiC.PROPERTY_MESSAGE,
+		TiC.PROPERTY_MESSAGEID,
 		TiC.PROPERTY_TITLE,
+		TiC.PROPERTY_TITLEID,
 		TiC.PROPERTY_OK,
+		TiC.PROPERTY_OKID,
```

Comment:
Thought about that, but the properties themselves have been available before, just the getter/setter haven't. So how would we document that particular case?